shims/common/src/main/scala/org/apache/gluten/GlutenConfig.scala

```diff
@@ -1585,6 +1585,12 @@ object GlutenConfig {
       .booleanConf
       .createWithDefault(false)
 
+  val DEBUG_KEEP_JNI_WORKSPACE_DIR =
+    buildConf(GLUTEN_DEBUG_KEEP_JNI_WORKSPACE_DIR)
+      .internal()
+      .stringConf
+      .createWithDefault("/tmp")
```

Comment:
I think we can just make the conf static, as well as `DEBUG_KEEP_JNI_WORKSPACE`. Thanks.